iis/build_release_amd64.bat

```diff
@@ -0,0 +1,50 @@
+
+echo "Building release..."
```

Comment:
is it wired if we only add this release amd amd64 but not but release x86 script? or we can make build_release take args and set the default to build all.

Reply:
Since we only build on x64 platforms, I didn't want to try and complicate the existing build script and hence just added a new one. If there is a strong argument for supporting x86 as an optional argument we can look at it.